### PR TITLE
RenderBase: clean up UpdateDrawRectangle

### PR DESCRIPTION
**Pull request by ligfx**
Extracted from #4992, as it's pretty orthogonal.

### CODE REVIEW

File: Source/Core/VideoCommon/RenderBase.h [hunk 1]
```diff
@@ -92,8 +92,9 @@ class Renderer
   // Use this to convert a whole native EFB rect to backbuffer coordinates
   virtual TargetRectangle ConvertEFBRectangle(const EFBRectangle& rc) = 0;
 
-  const TargetRectangle& GetTargetRectangle() const { return m_target_rectangle; }
-  float CalculateDrawAspectRatio(int target_width, int target_height) const;
+  const TargetRectangle& GetTargetRectangle() { return m_target_rectangle; }
```
Comment (lioncash) [inline Source/Core/VideoCommon/RenderBase.h]: Why did this lose the const specifier?

Comment (ligfx) [inline Source/Core/VideoCommon/RenderBase.h]: Cause rebasing 😥 Fixed.

**Comment by ligfx** [comment]
@degasus @stenzek @phire Could you please take a look at this when you have the time?

**Comment by iwubcode** [comment]
This cleanup looks good to me but I wasn't sure how to test it?

**Comment by ligfx** [comment]
@iwubcode I tested it by displaying frames in a window that was too tall (so they were centered vertically) and a window that was too wide (so they were centered horizontally), for both 16:9 and 4:3 aspect ratios. The displayed frame was the same size and in the same position as current master for each of the cases.

**Pull request merged**